src/ToggleButton.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" context="module">
+  import { capitalize } from "@app/utils";
+
+  export interface ToggleButtonOption<T> {
+    count?: number;
```

Comment:
When would `count` be `undefined`? It seems to me that we always define something as count.

Reply:
This is still a draft. I still wanted to refactor the tabs on the patch page, and they don't need a counter.
<img width="251" alt="Screenshot 2022-08-30 at 10 21 07" src="https://user-images.githubusercontent.com/158411/187387294-94f277a5-3eeb-4c0f-a74e-3f07ce6f0baa.png">

Reply:
Yeah sorry, looked into the PR as said to see your approach. Will leave you to it, surely an improvement over the current way